src/pal/prebuilt/idl/clrdata_i.cpp

```diff
@@ -14,15 +14,13 @@
 
 #pragma warning( disable: 4049 )  /* more than 64k source lines */
 
+#include <rpc.h>
```

Comment:
Would extern "C++" around the template stuff in palrt.h it is complaining about fix the issue? Then no changes to the MIDL generated files should be needed. Keeping those files as .C would likely be problematic, since there are wide strings in the interfaces defined in some of those files.